GameEngine/Characters/PlayerCharacter.cs

```diff
@@ -13,8 +13,12 @@ public PlayerCharacter(string name, int hp) : base(name, hp)
 
         internal void InternalTurn(EngineInternal engine)
         {
+            Console.Clear();
             string input;
-            this.SendMessage(">", false);
+            this.SendMessage($"{Name}:", false);
+            Console.WriteLine("");
+            PublicCommandHelper.TryRunPublicCommand("look", new List<string>(), this);
```

Comment:
This seems odd, after a character moves they also look.. but this is before the command.

Reply:
I will remove that look after moving